pkg/frontend/transport/handler.go

```diff
@@ -237,6 +237,7 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 		"fetched_index_bytes", numIndexBytes,
 		"sharded_queries", stats.LoadShardedQueries(),
 		"split_queries", stats.LoadSplitQueries(),
+		"fetched_series_estimate", stats.GetFetchedSeriesEstimate(),
```

Comment:
[nit] I suggest to call it "estimated series" can we have series we estimated and actual series we fetched.

```suggestion
		"estimated_series", stats.GetFetchedSeriesEstimate(),
```